packages/nextjs/components/builders/BuildersTable.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import Link from "next/link";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+type Props = {
+  builders: string[];
```

Comment:
This parameter name is confusing. It shouldn’t represent all builders, only those who have a profile page.